src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/SendNotificationHandler.java

```diff
@@ -125,12 +125,14 @@ private List<Event> getEventsToHandle(Callback<AsylumCase> callback) {
         if (!isPaid(callback)) {
             eventsToHandle.add(Event.END_APPEAL_AUTOMATICALLY);
         }
-        if (HandlerUtils.isAipJourney(callback.getCaseDetails().getCaseData()) && isPaid(callback)) {
+        if (isAipJourney(callback.getCaseDetails().getCaseData()) && isPaid(callback)) {
```

Comment:
This isn't a real change, it's just the static _isAipJourney_ method being changed because it's imported with an add-on static import

Reply:
Thanks for making it clear